frontend/apps/web/src/mocks/index.tsx

```diff
@@ -1,64 +1,101 @@
 import { Colors } from '@filecoin/ui'
 
-export const ChartData = [
+export const ChartDataWithoutPercentage = [
   {
     color: Colors.green90,
     border: Colors.green90,
-    value: 22,
-    description: '35% Unit',
+    size: 22,
+    name: '35% Unit',
   },
   {
     color: Colors.green30,
     border: Colors.green30,
-    value: 22,
-    description: '25% Integration',
+    size: 22,
+    name: '25% Integration',
   },
   {
     color: Colors.yellow,
     border: Colors.yellow,
-    value: 6,
-    description: '15% API',
+    size: 6,
+    name: '15% API',
   },
   {
     color: Colors.warningNotification,
     border: Colors.warningNotification,
-    value: 10,
-    description: '10% CLI',
+    size: 10,
+    name: '10% CLI',
   },
   {
     color: Colors.red90,
     border: Colors.red90,
-    value: 30,
-    description: '8% E2E',
+    size: 30,
+    name: '8% E2E',
   },
   {
     color: Colors.inputHoverPlaceholder,
     border: Colors.inputHoverPlaceholder,
-    value: 30,
-    description: '7% unkown',
+    size: 30,
+    name: '7% unkown',
   },
 ]
 
+export const chartDataWithPercentageAndKind = [
+  {
+    kind: 'api',
+    percentage: 14.9
+  },
+  {
+    kind: 'unit',
+    percentage: 49
+  },
+  {
+    kind: 'integration',
+    percentage: 22.8
+  },
+  {
+    kind: 'benchmark',
+    percentage: 8
+  },
+  {
+    kind: 'e2e',
+    percentage: 6
+  }
+]
+
+export const chartDataWithPercentageAndStatus = [
+  {
+    kind: 'api',
+    percentage: 14
```

Comment:
percentage should always be 100% total. and use more decimals so that we know what will happen in scenario from the screenshot. I guess that in Legend we want to show only two decimals, etc...

Reply:
Done